### PR TITLE
Fixed descriptions and usages while using the help command

### PR DESCRIPTION
**Pull request by jonasjac**

### CODE REVIEW

File: src/commands/ban.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = class KickCommand extends GenericModCommand {
       action: ['ban', 'banned'],
       dm: true,
       triggers: ['ban'],
-      usage: 'no uuu',
-      description: 'hiiii'
+      usage: 'ban [@user] <reason>',
+      description: 'Bans the mentioned user'
```
Comment (Aetheryx) [inline src/commands/ban.js]: should be the other way around - `<user> [reason]`, not `[user] <reason>`

File: src/commands/upload.js [hunk 2]
```diff
@@ -33,8 +33,8 @@ module.exports = class UploadCommand {
   get props () {
     return {
       triggers: ['upload'],
-      usage: 'no',
-      description: 'stop that'
+      usage: 'upload <image>',
+      description: 'Uploads your image to a host'
```
Comment (Aetheryx) [inline src/commands/upload.js]: image isn't really an argument here, it's a file attachment. Also might be worthwhile to specify the host itself, https//boisbetterthanluca.xyz

**Comment by Aetheryx** [comment]
You weren't supposed to remove `{command}` from all of the usages, that's a template string. It's replaced by `$PREFIX $COMMAND` in the help command. All of the mod commands also have the argument priority reversed, it's `<user> [reason]`, not `[user] <reason>`.

**Comment by 21Joakim** [comment]
Newb

**Pull request merged**